weather.noaa/default.py

```diff
@@ -0,0 +1,1204 @@
+# -*- coding: utf-8 -*-
```

Comment:
I stronlgy recomment to move all this code to an importable module leaving only a minimax entrypoint code. This will speed-up addon execution on lower-end systems like single-board computers because bytecode of imported modules is cached.